sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -56,7 +56,7 @@ For Metrics queries across multiple Azure resources, use the following client:
 
 ```C# Snippet:CreateMetricsClient
 var client = new MetricsClient(
-    new Uri("https://metrics.monitor.azure.com/.default"),
+    new Uri("https://<region>.metrics.monitor.azure.com/"),
```

Comment:
Do we not need default anymore?

Reply:
The .default is needed for the scope so adding it here made the snippet incorrect :)